src/components/Users/TwoFactorAuth.tsx

```diff
@@ -168,7 +168,7 @@ export const TwoFactorAuth = ({ userData }: userChildProps) => {
           <div className="space-y-3">
             {!userData.mfa_enabled ? (
               <>
-                <p className="text-sm text-gray-700">
+                <p className="text-sm text-gray-700 text-center sm:text-left">
```

Comment:
```suggestion
                <p className="text-sm text-gray-700 break-normal hyphens-none">
```
current implementation does not work and only aligns text to middle, still there is breakage in text